test/ubuntu/submodules/ubuntu/10_containers.tf

```diff
@@ -37,7 +37,7 @@ locals {
           path    = "/"
           port    = "none"
           host    = ""
-          scheme  = "http"
+          scheme  = "HTTP"
```

Comment:
could you please also adjust this and the other ocurences here in [example/module/submodules/replaceMe/10_containers.tf](https://github.com/sers-dev/terraform-k8s-template/blob/main/example/module/submodules/replaceMe/10_containers.tf)